docs/administering/offline.md

```diff
@@ -0,0 +1,100 @@
+This document explains how to run a Sandstorm server that has no access to the internet ("airgapped" or "offline"). Please note that running such a server requires a bit of work, but it can be done.
+
+## Installing Sandstorm Offline
+
+By default, Sandstorm's install script downloads the latest release from `sandstorm.io`, but the script can also accept a release tarball as a parameter. To use it offline, you will need to obtain the installer script and a release tarball from the internet and transfer them to your server.
+
+You can find and download the latest release by going to: [https://dl.sandstorm.io/](https://dl.sandstorm.io/) Look for the files titled `sandstorm-<version>.tar.gz` and find the one with the highest number for `<version>`. Download this file.
+
+You can obtain the latest version of the installer script from our GitHub repository: [install.sh](https://raw.githubusercontent.com/sandstorm-io/sandstorm/master/install.sh) Transfer this file to your server as well.
+
+Now, on your server, you can run:
+
+    bash install.sh sandstorm-<version>.tar.gz
+
+(Replace `<version>` with the version number you downloaded.)
+
+Follow the on-screen instructions to set up your server. Note that:
+
+- You cannot use the Sandcats DNS service. You must configure your own DNS. Remember that [Sandstorm requires a wildcard DNS entry](wildcard.md).
+- You must use LDAP or SAML login to integrate with your network's single sign-on. Google, Github, and (probably) e-mail login will not work since they require internet access.
+- When you reach the step in the installer where Sandstorm wants to pre-install some apps, choose "Skip for now", since Sandstorm won't be able to reach the app server.
+
+## Updating Offline
+
+Normally, Sandstorm automatically updates itself. Of course, it cannot do this if it does not have access to the internet. You should disable automatic updates by editing `/opt/sandstorm/sandstorm.conf`. Find the line for `UPDATE_CHANNEL` and change it to:
+
+    UPDATE_CHANNEL=none
+
+Then restart Sandstorm:
+
+    sudo sandstorm stop
+    sudo sandstorm start
+
+Now, to update Sandstorm manually, obtain the latest release tarball from `dl.sandstorm.io` the same as you did to install it originally. Upload this to your server.
+
+Then, run:
+
+    sudo sandstorm update sandsotrm-<version>.tar.gz
+
+## Installing Apps
+
+To install an app, you must manually obtain the app's SPK package file, then upload it to your server through its web interface. To obtain an SPK package from the app market, [visit the market](https://apps.sandstorm.io), find the app you want, and click the "Download SPK" button in the upper-right of the app's page. This will download the SPK file.
```

Comment:
awkward pronoun: "upload it to your server through its web interface" — the package's web interface?